src/app/state/GameContext.tsx

```diff
@@ -16,6 +16,20 @@ interface Props {
   connection: Connection;
   player: Player;
 }
+const [GameProvider, useGame] = createUseContext(() => {
```

Comment:
Har det blitt en merge feil her? Vi lager provideren to ganger ser det ut til.

Reply:
Fikser!